app/models/export/submissions/row.rb

```diff
@@ -0,0 +1,100 @@
+module Export
+  class Submissions
+    class Row
+      ERROR   = '#ERROR'.freeze
+      MISSING = '#MISSING'.freeze # fields that are needed for MVP that we don't have yet
+
+      attr_reader :submission, :errors
+
+      def initialize(submission)
+        @submission = submission
+        @errors = Hash.new { |hash, column_name| hash[column_name] = [] }
+      end
+
+      def row_values
+        [
+          submission.task_id,
+          submission.id,
+          status,
+          submission_type,
+          submission_file_type,
+          order_entry_count,
+          order_value,
+          invoice_entry_count,
+          invoice_value,
+          management_charge_value,
+          management_charge_rate,
+          created_date,
+          created_by,
+          supplier_approved_date,
+          supplier_approved_by,
+          finance_export_date,
+          po_number
+        ]
+      end
+
+      def status
+        {
+          'completed'         => 'supplier_accepted',
+          'validation_failed' => 'validation_failed'
+        }.fetch(submission.aasm_state)
+      rescue KeyError
+        errors['Status'] << "#{submission.aasm_state} is not mapped to Submission column Status"
+        ERROR
+      end
+
+      def submission_type
+        (invoice_entry_count + order_entry_count).zero? ? 'no_business' : 'file'
+      end
+
+      def submission_file_type
+        File.extname(submission._first_filename).downcase[1..-1]
+      end
+
+      def management_charge_value
+        submission.management_charge
```

Comment:
The spec is definitely vague. The correct output should be in pounds and pence. So a management charge of £129.12 would be represented as 129.12 rather than 12912.

Reply:
`submissions.management_charge` is an `Integer`, so I'm guessing internal to DataSubmissionServiceAPI we're storing in pence?

Reply:
@rgarner correct